src/utils/daos_hdlr.c

```diff
@@ -712,20 +712,19 @@ cont_check_hdlr(struct cmd_args_s *ap)
 		}
 	}
 
-	daos_gettime_coarse(&end);
-
+	end = time(NULL);
 	duration = end - begin;
 	if (duration == 0)
 		duration = 1;
```

Comment:
Not a big deal, but I suppose most test run can finish within few milli-seconds? which is far less than 1 second. keeping the duration as 0 in that case looks more reasonable to me.

Reply:
It is just some coarse-grained calculation.